File: matiasmasso/Mat.NET/DTO/DTOCodiMercancia.cs
namespace DTO
{
    public class DTOCodiMercancia
    {
        private static string EmptyValue = new string('0', 8);

        public string Id { get; set; }
        public string Dsc { get; set; }

        public bool IsLoaded { get; set; }

        public DTOCodiMercancia(string id) : base()
        {
            Id = id;
        }

        public static string FullNom(DTOCodiMercancia oCodiMercancia)
        {
            string retval = "";
            if (oCodiMercancia != null)
                retval = string.Format("{0} {1}", oCodiMercancia.Id, oCodiMercancia.Dsc);
            return retval;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOCodiMercancia.vb
Public Class DTOCodiMercancia
    Shared EmptyValue As New String("0", 8)

    Property Id As String
    Property Dsc As String

    Property IsLoaded As Boolean

    Public Sub New(Id As String)
        MyBase.New
        _Id = Id
    End Sub

    Shared Function FullNom(oCodiMercancia As DTOCodiMercancia) As String
        Dim retval As String = ""
        If oCodiMercancia IsNot Nothing Then
            retval = String.Format("{0} {1}", oCodiMercancia.Id, oCodiMercancia.Dsc)
        End If
        Return retval
    End Function
End Class
